packages/core/components/FileDetails/index.tsx

```diff
@@ -78,14 +78,20 @@ function resizeHandleDoubleClick() {
 export default function FileDetails(props: Props) {
     const [windowState, windowDispatch] = React.useReducer(windowStateReducer, INITIAL_STATE);
     const [fileDetails, isLoading] = useFileDetails();
+    const [thumbnailPath, setThumbnailPath] = React.useState<string | undefined>(undefined);
```

Comment:
`undefined` is implicit
```suggestion
    const [thumbnailPath, setThumbnailPath] = React.useState<string | undefined>();
```

Reply:
resolved in [552170f](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/143/commits/552170f1bcfa6f438376984f8f4c54127cff0db5)[552170f](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/143/commits/552170f1bcfa6f438376984f8f4c54127cff0db5)[552170f](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/143/commits/552170f1bcfa6f438376984f8f4c54127cff0db5)